### ISSUE TITLE
NM crashing my game - and constantly running at retail speed on just these 2 characters

### ISSUE THREAD

**Issue by Devolnu** [HomepointXI/Issues#146]
**Date & Time**:  3:04 PM 7/21/2018

**Client Version (use `/ver` in game):** up to date

**Character Name:** Devolnu and Elbavolnu

**Discord Name (if different from in game):**

**Nation:** Bastok

**Job(level)/Sub Job(level):**

**NPC or Monster or item Name:** All NMs

**Zone name:** Any

**Coordinates (use `!where`):** All

**Item tool link (for items issues only):**

**Multi-boxing? (multiple clients on same connection):** No

**Steps to reproduce:**
Killed an NM crashed, signed back in killed another crashed.
speed is random on each sign in

**Any additional information:**

**Comment by spectre99** [comment]
I did some testing with Dev.  At first I thought it was client based, asked him to log on with Dev and Elb on skittles machine.  He did and killed a nm, characters crashed.  I also jailed and freed Dev again, when he came out his run speed was retail for some reason.  After that I gave him a login/pw for a character he didn't have access to before, went out and killed a nm, no crash.  Then Dev logged on with another character he didn't have access to before, killed a nm, everything was fine.  Then I had him stay logged in with someone elses char, and log in Dev, fought a nm with both characters next to one another, only Dev crashed.  The run speed thing seems random, every so often he logs on and chars run at retail speed, but the crashing on Dev and Elb seem constant no matter what machine those characters are on.
